pkg/exchange/okex/convert.go

```diff
@@ -306,3 +285,49 @@ func toGlobalOrder(okexOrder *okexapi.OrderDetails) (*types.Order, error) {
 		IsIsolated:       false,
 	}, nil
 }
+
+func toGlobalTrade(orderDetail *okexapi.OrderDetails) (*types.Trade, error) {
+	tradeID, err := strconv.ParseInt(orderDetail.LastTradeID, 10, 64)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error parsing tradeId value: %s", orderDetail.LastTradeID)
+	}
+
+	orderID, err := strconv.ParseInt(orderDetail.OrderID, 10, 64)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error parsing ordId value: %s", orderDetail.OrderID)
+	}
+
+	side := toGlobalSide(orderDetail.Side)
+
+	isMargin := false
+	if orderDetail.InstrumentType == string(okexapi.InstrumentTypeMARGIN) {
```

Comment:
can you replace the type of the field in the orderDetail with `InstrumentType` ? so that you don't have to convert them into string() ?

Reply:
Updated ~